both/routes.js

```diff
@@ -53,7 +52,7 @@ if(Meteor.isClient) {
     // previous (old) page. Waiting for a little bit before setting the title
     // lets the browser history look sane.
     setTimeout(function() {
-      document.title = title;
+      document.title = titleParts.reverse().join(' - ');
```

Comment:
Nice cleanup. Is there a reason you `push()`-ed and `reverse()`-ed instead of `unshift()`-ing and saving the reverse at the end?